src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -67,7 +67,13 @@ public static CommitResult analyzeCommit(CommitInfo commitInfo, Map<String, Auth
             logger.log(Level.WARNING, "Unable to parse the date from git log result for commit.", pe);
         }
 
-        String message = elements[MESSAGE_INDEX];
+        String message = "";
+        try {
```

Comment:
will using `elements.length` to do a check instead of catching the exception be a better solution?

Reply:
Yes, we typically only use `try/catch` of exceptions for situations that are 'exceptional'/out of our control.

In this case, an empty commit message isn't exactly 'exceptional', but just 1 special/corner case. As such, it is preferred to use a `if` statement to check for the array's length, and setting the message to be empty if it is out of range. We also don't need a logging message for this case as well.

Reply:
Thanks  for pointing out. Have updated accordingly.